### PR TITLE
Azure Backup One SDK 

### PR DESCRIPTION
**Pull request by swatimadhukar**
Azure Backup One SDK Changes for remaining backup operations which includes Item, protection, jobs, container, protection policy, backup and restore.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@swatimadhukar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.Azure.Management.BackupServices;
```
Comment (yugangw-msft) [inline locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs]: license header

File: locker/AzureBackup/BackupServices.Tests/ScenarioTests/AzureBackupItemTests.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
+﻿//
```
Comment (yugangw-msft) [inline locker/AzureBackup/BackupServices.Tests/ScenarioTests/AzureBackupItemTests.cs]: please do not put the folder under root, unless you have good reason.

File: src/ResourceManagement/AzureBackup/BackupServices.Tests/app.config [hunk 3]
```diff
@@ -1,30 +1,59 @@
...
-      </providers>
-    </membership>
-    <roleManager defaultProvider="ClientRoleProvider" enabled="true">
-      <providers>
-        <add name="ClientRoleProvider" type="System.Web.ClientServices.Providers.ClientRoleProvider, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" serviceUri="" cacheTimeout="86400" />
-      </providers>
-    </roleManager>
-  </system.web>
+	<appSettings>
+		<add key="ResourceName" value="backuprn" />
```
Comment (yugangw-msft) [inline src/ResourceManagement/AzureBackup/BackupServices.Tests/app.config]: please keep the original indent

File: locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs (hunk 1 above)
Comment (yugangw-msft) [inline locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs]: please do the same for other files

**Comment by yugangw-msft** [comment]
please increase version number in the nuget proj file

File: locker/AzureBackup/BackupServices.Tests/ScenarioTests/AzureBackupItemTests.cs (hunk 2 above)
Comment (swatimadhukar) [inline locker/AzureBackup/BackupServices.Tests/ScenarioTests/AzureBackupItemTests.cs]: Can you please explain how you want the folder structure to be ?

File: src/ResourceManagement/AzureBackup/BackupServices.Tests/app.config (hunk 3 above)
Comment (swatimadhukar) [inline src/ResourceManagement/AzureBackup/BackupServices.Tests/app.config]: We have done VS indentation this time. Going forward we will be using VS indentation.

File: locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs (hunk 1 above)
Comment (swatimadhukar) [inline locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs]: The locker folder got added by mistake by one of my teammate. I have removed it.

**Pull request merged**